src/AirBench/Api/Controllers/BenchController.cs

```diff
@@ -0,0 +1,173 @@
+﻿using AirBench.Api.Models;
+using AirBench.Api.Repositories;
+using AirBench.Models;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace AirBench.Api.Controllers
+{
+    public class BenchController : ApiController
+    {
+        private IBenchApiRepository _benchRepo;
+
+        public BenchController(IBenchApiRepository benchRepo)
+        {
+            _benchRepo = benchRepo;
+        }
+
+        /// <summary>
+        /// Attempts to add a bench.
+        /// 
+        /// ROUTE:
+        /// "bench/add"
+        /// 
+        /// REQUEST BODY:
+        /// {
+        ///     "Description": `string`,
+        ///     "Latitude": `float`,
+        ///     "Longitude": `float`,
+        ///     "NumberSeats": `int`
+        /// }
+        /// 
+        /// RESPONSE BODY:
+        /// {
+        ///     "Success": `bool`,
+        ///     "Id": `int`,
+        ///     "Description": `string`,
+        ///     "Latitude": `float`,
+        ///     "Longitude": `float`,
+        ///     "NumberSeats": `int`
+        /// }
+        /// </summary>
+        /// <param name="request">The request body.</param>
+        /// <returns>The response body.</returns>
+        [HttpPost]
+        public async Task<BenchAddResponse> Add(BenchAddRequest request)
+        {
+            var bench = new Bench()
+            {
+                Description = request.Description,
+                Latitude = request.Latitude,
+                Longitude = request.Longitude,
+                NumberSeats = request.NumberSeats
+            };
+            var benchId = await _benchRepo.AddAsync(bench);
```

Comment:
What happens when the HTTP post request doesn't have values for each of the `BenchAddRequest` properties?